src/Arcus.Messaging.Pumps.ServiceBus/Configuration/AzureServiceBusMessagePumpOptions.cs

```diff
@@ -104,12 +104,13 @@ public int MaximumUnauthorizedExceptionsBeforeRestart
         /// <summary>
         /// Gets the consumer-configurable options to change the deserialization behavior.
         /// </summary>
-        public MessageDeserializationOptions Deserialization => MessageRouterOptions.Deserialization;
+        [Obsolete("Use the " + nameof(Routing) + ".Deserialization instead")]
+        public MessageDeserializationOptions Deserialization => Routing.Deserialization;
```

Comment:
Deprecation message not present when configuring the message pump, as they are using interfaces to provide the correct set of options. This deprecation message is more for contributors and consumers that want to write custom message routers.